code/modules/experisci/experiment/handlers/experiment_handler.dm

```diff
@@ -56,6 +56,8 @@
 		RegisterSignal(parent, COMSIG_DOPPLER_ARRAY_EXPLOSION_DETECTED, .proc/try_run_doppler_experiment)
 	if(istype(parent, /obj/machinery/destructive_scanner))
 		RegisterSignal(parent, COMSIG_MACHINERY_DESTRUCTIVE_SCAN, .proc/try_run_destructive_experiment)
+	if(istype(parent, /obj/machinery/computer/operating))
```

Comment:
For the record, this feels like a really weird God component. Project for another day.